pwa/src/components/common/layout.tsx

```diff
@@ -20,11 +20,13 @@ import favicon from "../../images/conduction_logo_blauw.svg";
  * @returns TSX of the generated Layout.
  */
 export default function Layout({ children, title = "", subtext = "" }) {
+  const [jwt, setJwt] = React.useState<string>(null);
   const [API, setAPI] = React.useState<APIService>(null);
 
   React.useEffect(() => {
-    !API && setAPI(new APIService(sessionStorage.getItem("jwt")));
-  }, [API]);
+    !jwt && setJwt(sessionStorage.getItem("jwt"));
+    !API && jwt && setAPI(new APIService(sessionStorage.getItem("jwt")));
+  }, [API, jwt]);
 
   return (
     API && (
```

Comment:
Volgens mij zie ik een wit scherm omdat je hier alleen layout returned als er een API is en die maak je vgm alleen aan als je een jwt hebt die je pas krijgt als je ingelogt bent